### ISSUE TITLE
The parameters of loss are always similarity during training

### ISSUE THREAD

**Issue by Leputa** [Mikoto10032/AutomaticWeightedLoss#11]
I have 3 losses, such as 
"""
tensor([0.9926, 0.9926, 0.9927], requires_grad=True)
tensor([0.9908, 0.9908, 0.9909], requires_grad=True)
tensor([0.9873, 0.9873, 0.9873], requires_grad=True)
"""

**Comment by long123524** [comment]
我也是这样的，请问你解决了吗？

**Comment by timerobin** [comment]
I  meet the same problem two

**Comment by zhiqihuang** [comment]
Same issue. Any idea?
